packages/amplify-container-hosting/lib/ElasticContainer/index.js

```diff
@@ -0,0 +1,308 @@
+        envName,
+        categoryName,
+        apiName: resourceName,
+        authName,
+        dependsOn,
+        policies: [], // TODO
+        deploymentBucketName,
+        restrictAccess,
+        createCloudMapService: false,
+        secretsArns,
+        isInitialDeploy,
+        deploymentMechanism: resource.deploymentMechanism,
+        domainName: resource.domain,
+        desiredCount,
+        hostedZoneId,
+        containers,
+        exposedContainer,
+        taskPorts: containersPorts,
+        gitHubSourceActionInfo: undefined,
+        taskEnvironmentVariables: {}, // TODO
+        existingEcrRepositories,
+    });
+
+    context.exeInfo.template = stack.toCloudFormation();
+
+    const resourceDirPath = path.join(projectBackendDirPath, constants.CategoryName, serviceName);
+    fs.ensureDirSync(resourceDirPath);
+
+    const templateFilePath = path.join(resourceDirPath, templateFileName);
+    let jsonString = JSON.stringify(context.exeInfo.template, null, 4);
+    fs.writeFileSync(templateFilePath, jsonString, 'utf8');
+
+    if (addResource) {
+        return context.amplify.updateamplifyMetaAfterResourceAdd(constants.CategoryName, serviceName, resource);
+    } else {
+        await context.amplify.updateamplifyMetaAfterResourceUpdate(constants.CategoryName, serviceName, 'exposedContainer', exposedContainer);
+    }
+}
+
+function CheckIsValidDomain(domain) {
+    var re = new RegExp(/^(\*\.)?(((?!-)[A-Za-z0-9-]{0,62}[A-Za-z0-9])\.)+((?!-)[A-Za-z0-9-]{1,62}[A-Za-z0-9])$/);
+    const validDomain = re.test(domain);
+
+    return validDomain || `Domain ${domain} invalid`;
+}
+
+async function configure(context) {
+
+}
+
+async function publish(context, args) {
+    const { frontend } = context.amplify.getProjectConfig();
+    const { config: { SourceDir: src, DistributionDir: dst } } = context.amplify.getProjectConfig()[frontend];
+
+    const { projectPath } = context.amplify.getEnvInfo();
+    const srcPath = path.join(projectPath, src);
+    const fileName = 'site.zip';
+    const filePath = path.join(dst, fileName)
+
+    await context.amplify.executeProviderUtils(context, 'awscloudformation', 'zipFiles',
+        [srcPath, filePath]);
+
+    const {
+        providers: { [constants.providerName]: provider },
+    } = context.amplify.getProjectMeta();
+    const { DeploymentBucketName: deploymentBucketName } = provider;
+
+
+    const s3Client = await getS3Client(context, 'publish');
+    await uploadFile(s3Client, deploymentBucketName, filePath, fileName)
+
+    const { hosting: { ElasticContainer: { output: { PipelineUrl } } } } = context.amplify.getProjectMeta();
+
+    context.print.info(`\nPublish started, you can check the status of the deployment on:\n${PipelineUrl}`);
+
+}
+
+function console(context) {
+    // Check this behavior
+    const consoleUrl = `xxxx`;
+    context.print.info(consoleUrl);
+    //   open(consoleUrl, { wait: false });
+}
```

Comment:
either throw error or implement this

Reply:
This will be implemented, the commit is coming today